src/app/searchsettings/searchsettings.component.spec.ts

```diff
@@ -1,14 +1,68 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { Component, Input, Output } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+import { NavbarComponent } from '../navbar/navbar.component';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+import { HttpModule, JsonpModule } from '@angular/http';
+import { StoreModule } from '@ngrx/store';
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { AppComponent } from '../app.component';
+import { IndexComponent } from '../index/index.component';
+import { ResultsComponent } from '../results/results.component';
+import { NotFoundComponent } from '../not-found/not-found.component';
+import { AdvancedsearchComponent } from '../advancedsearch/advancedsearch.component';
+import { SearchBarComponent } from '../search-bar/search-bar.component';
+import { reducer } from '../reducers/index';
+import { FooterNavbarComponent } from '../footer-navbar/footer-navbar.component';
+import { AboutComponent } from '../about/about.component';
+import { ModalComponent } from 'ng2-bs3-modal/ng2-bs3-modal';
+import { ContactComponent } from '../contact/contact.component';
+import {InfoboxComponent} from "../infobox/infobox.component";
+import { RelatedSearchComponent } from '../related-search/related-search.component';
+import {KnowledgeapiService} from "../knowledgeapi.service";
+import {AutocompleteService} from "../autocomplete.service";
+import {AutoCompleteComponent} from "../auto-complete/auto-complete.component";
+import { ThemeComponent } from '../theme/theme.component';
+import { DropdownComponent } from '../dropdown/dropdown.component';
 import { SearchsettingsComponent } from './searchsettings.component';
+import {BrowserModule} from "@angular/platform-browser";
 
 describe('SearchsettingsComponent', () => {
   let component: SearchsettingsComponent;
   let fixture: ComponentFixture<SearchsettingsComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ SearchsettingsComponent ]
+      imports: [
+        RouterTestingModule,
+        BrowserModule,
+        CommonModule,
+        FormsModule,
+        HttpModule,
+        JsonpModule,
+        StoreModule.provideStore(reducer),
+        StoreDevtoolsModule.instrumentOnlyWithExtension(),
+      ],
+      declarations: [
+        AppComponent,
+        NavbarComponent,
+        IndexComponent,
+        ResultsComponent,
+        NotFoundComponent,
+        AdvancedsearchComponent,
+        SearchBarComponent,
+        FooterNavbarComponent,
+        AboutComponent,
+        ContactComponent,
+        ModalComponent,
+        InfoboxComponent,
+        RelatedSearchComponent,
+        AutoCompleteComponent,
+        ThemeComponent,
+        DropdownComponent,
+        SearchsettingsComponent
```

Comment:
![image](https://user-images.githubusercontent.com/15216503/27765090-a8de71d0-5ec6-11e7-8e44-e4a6160758ea.png)

add IntelligenceComponent here this will solve the issue.